### PR TITLE
Traverse Topology in DFS format in block topology output

### PR DESCRIPTION
**Pull request by ritikasrivastava**

### CODE REVIEW

**Comment by henryh2** [comment]
Perhaps this would be a good MR to introduce a document in repo that explains the vertex structure, just to make sure everyone is on the same page. I think that might be pretty helpful, perhaps we should discuss during review?

File: pkg/translate/output.go [hunk 1]
```diff
@@ -111,41 +111,69 @@ func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
 	return strconv.Itoa(int(bs))
 }
 
-func toBlockTopology(wr io.Writer, root *topology.Vertex) error {
-	// traverse tree topology and when a node is reached, check within blockRoot for domain and print that domain.
+/*
```
Comment (dmitsh) [inline pkg/translate/output.go]: Do we need to keep the commented-out code?

File: pkg/translate/output.go [hunk 2]
```diff
@@ -111,41 +111,69 @@ func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
...
+		blockSize := ""
+		if _, exists := root.Metadata[topology.KeyBlockSizes]; exists {
+			blockSize = root.Metadata[topology.KeyBlockSizes]
+		}
+		blockSize = getBlockSize(domainVisited, blockSize)
+		_, err = wr.Write([]byte(fmt.Sprintf("BlockSizes=%s\n", blockSize)))
+		return err
+	}
+*/
+func toBlockTopologyDFS(wr io.Writer, root *topology.Vertex) error {
```
Comment (dmitsh) [inline pkg/translate/output.go]: are we planning to have other types of `toBlockTopology` beside DFS? If not, perhaps we could keep `toBlockTopology()`?

File: pkg/translate/output.go (hunk 1 above)
Comment (ritikasrivastava) [reply pkg/translate/output.go]: No, we do not. Just for review purposes. I can remove it once everyone got a chance to review.

File: pkg/translate/output.go (hunk 2 above)
Comment (ritikasrivastava) [reply pkg/translate/output.go]: No, we do not. Just for review purposes. I can rename it

**Review by dmitsh — APPROVED** [review]

**Review by dmitsh — APPROVED** [review]

**Pull request merged**